e2e/packages/client-vanilla/src/mud/setupNetwork.ts

```diff
@@ -38,7 +38,7 @@ export async function setupNetwork() {
     address: networkConfig.worldAddress as Hex,
     publicClient,
     startBlock: BigInt(networkConfig.initialBlockNumber),
-    indexerUrl: networkConfig.indexerUrl ?? undefined,
+    indexerUrl: networkConfig.indexerUrl,
```

Comment:
I think this was intentionally here to make types happy because `params.get` can return null